README.md

```diff
@@ -19,66 +19,67 @@ We use `main` branch as the develop branch while setting the default branch as t
 |---------------------------|----------------|----------------|
 | main                      | false          | develop branch |
 | spring-cloud-azure_v4.0.0 | true           | release branch |
-
+| spring-cloud-azure_v4.1.0 | true           | release branch |
 
 ## Current Branch Supported Versions
-- [spring-boot-dependencies:2.6.3](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.3/spring-boot-dependencies-2.6.3.pom).
-- [spring-cloud-dependencies:2021.0.1](https://repo.maven.apache.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.1/spring-cloud-dependencies-2021.0.1.pom).
+- [spring-boot-dependencies:2.6.7](https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.7/spring-boot-dependencies-2.6.7.pom).
+- [spring-cloud-dependencies:2021.0.2](https://repo.maven.apache.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.2/spring-cloud-dependencies-2021.0.2.pom).
 
 ## All Active Branches
 
-| Spring Boot Version  | Spring Cloud version  | Spring Cloud Azure Version                                                                 |
-|----------------------|-----------------------|--------------------------------------------------------------------------------------------|
-| 2.6.3                | 2021.0.1              | [4.0.0](https://github.com/Azure/azure-sdk-for-java/releases/tag/spring-cloud-azure_4.0.0) | 
+| Spring Boot Version | Spring Cloud version | Spring Cloud Azure Version                                                                 |
+|---------------------|----------------------|--------------------------------------------------------------------------------------------|
+| 2.6.3               | 2021.0.1             | [4.0.0](https://github.com/Azure/azure-sdk-for-java/releases/tag/spring-cloud-azure_4.0.0) | 
+| 2.6.7               | 2021.0.2             | [4.1.0](https://github.com/Azure/azure-sdk-for-java/releases/tag/spring-cloud-azure_4.1.0) | 
```

Comment:
It's necessary to add a new column about `azure-spring-boot-samples` branches:
![image](https://user-images.githubusercontent.com/13167207/168009168-c3788e5c-7dac-4000-bba4-c10b807f3d25.png)

Reply:
We can delete this table if we only have 1 active branch.